src/kixi/datastore/metadatastore.clj

```diff
@@ -51,9 +50,38 @@
 (s/def ::segmentations
   (s/cat :segmentations (s/+ ::segmentation)))
 
+(s/def ::valid boolean?)
+
+(s/def ::spec-explain
+  (s/keys))
```

Comment:
This just says that a spec-explain is a map, and makes no assertions about it. There must be an actual spec for them somewhere in the core...